hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1053,6 +1053,29 @@ public function upgrade_1043() {
     return TRUE;
   }
 
+  /**
+   * Creates option group for organisation contact belonging to sub types
+   * Health Insurance Provider, Life Insurance Provider, or Pension Provider
+   *
+   * @return bool
+   */
+  public function upgrade_1044() {
+    $result = civicrm_api3('OptionGroup', 'get', [
+      'name' => ['IN' => [
+        'hrjc_health_insurance_provider',
+        'hrjc_life_insurance_provider',
+        'hrjc_pension_providers'
```

Comment:
I noticed that the first two option group are singular while the third is plural. We should stick to one. Singular should work for all.